### DISCUSSION TITLE
BlockHandler List Tag

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#888] [Help] [answered]
Hello!

I have a barrel and i need a list of nbtcompounds like in this picture.

How can i achieve this in getBlockEntityTags() cause there isn't a Tag.NBTList element or something similar.

![grafik](https://user-images.githubusercontent.com/82600264/162252826-12ca1773-9cdc-4285-8a0c-d9d93864a719.png)

Thanks 😄 

Oh, and how to do it for an IntArray? im new to minestom :|

**Comment by MrlnHi** [comment, marked as answer]
If you want a list of say Strings you would do
```java
Tag.String("tag").list()
```

**Reply by DasLixou** [reply]
Genius! But what about an array? like an IntegerArray? also that?

**Reply by TheMode** [reply]
Alternatively you can use `Tag.NBT` using an IntegerArray directly. But I do not really see the point instead of an integer list?
